admin/lib/generators/solidus_admin/component/component_generator.rb

```diff
@@ -11,6 +11,11 @@ class ComponentGenerator < Rails::Generators::NamedBase
     class_option :js, type: :boolean, default: true
     class_option :spec, type: :boolean, default: true
 
+    def setup_inflections
+      # This is needed to ensure that UI is not rendered as Ui
+      SolidusAdmin::Engine.initializers.find { _1.name =~ /inflections/ }.run
```

Comment:
Can you help me understand why we need this one? Where is it consumed? :thinking:

Reply:
I see, thanks for the clarification!

Reply:
Another workaround that doesn't require to duplicate the code (but requires a specific file for the inflector initialization) could be to create a separate initializer file in the gem and require it in the generator. I think it might feel less hacky but I'm also ok with this solution, just wanted to point it out.